xiv_stats.php

```diff
@@ -399,18 +398,61 @@ function getValueOrZero($data, $key) {
               <div class="card white">
                   <div class="card-content black-text">
                       <span class="card-title black-text" style="font-size:28pt;">XIVCensus - Player statistics for FFXIV</span>
-                      <br/>Statistics for <?php echo $date; ?>
-                      <p><b>* (Any reference to "Active" players, refers to players who have had their profile image update in the last 4 weeks)</b></p>
-                      <p><b>NOTE: The metric for "Active" players changed between February and March 2017</b></p>
-                  </div>
+                      <p>Statistics for <?php echo $date; ?></p>
+                      <p><b>* (Any reference to "Active" players, refers to players that have claimed the 3.3 story minion)</b></p>
+                    </div>
               </div>
           </div>
       </div>
+      <div class="col s12 m6 center-align">
```

Comment:
Could the links be sticky'd to the top of the viewport as the screen scrolls? Otherwise you will find yourself scrolling all the way back to the top in order to get back to these links.